qiskit/optimization/problems/quadratic_program.py

```diff
@@ -152,36 +153,139 @@ def _add_variable(self,
                       upperbound: Union[float, int] = INFINITY,
                       vartype: VarType = VarType.CONTINUOUS,
                       name: Optional[str] = None) -> Variable:
-        """Checks whether a variable name is already taken and adds the variable to list and index
-        if not.
+        return self._add_variables(lowerbound, upperbound, vartype, name)[1][0]
+
+    def _add_variables(self,
+                       lowerbound: Union[float, int] = 0,
+                       upperbound: Union[float, int] = INFINITY,
+                       vartype: VarType = VarType.CONTINUOUS,
+                       name: Optional[str] = None,
```

Comment:
There's no need to have default values, the method is always called with `lb`, `ub`, `vartype`, and `name` specified.